README.md

```diff
@@ -1,2 +1,56 @@
-# VT-WebClient
-Web Client for V Transfer.
+# Web Client for V Transfer
+![JavaScript](https://img.shields.io/badge/Javascript-3DDC84?style=for-the-badge&logo=javascript&logoColor=white) [![Slack](https://img.shields.io/badge/Slack-4A154B?style=for-the-badge&logo=slack&logoColor=white)](https://join.slack.com/t/codegambit/shared_invite/zt-pe1nuhbk-iPuFm2B1JuMS86od4a4wXQ) [![License](https://img.shields.io/badge/License-GPL-lightgrey.svg?style=for-the-badge)](https://github.com/code-gambit/VT-WebClient/blob/development/LICENSE) ![GitHub code size in bytes](https://img.shields.io/github/languages/code-size/code-gambit/VT-WebClient?style=for-the-badge) <br>
+[![CircleCI Workflow Status](https://img.shields.io/circleci/build/gh/code-gambit/VT-WebClient.svg?style=for-the-badge)](https://github.com/code-gambit/VT-WebClient/blob/development/.circleci/config.yml)
+![React](https://img.shields.io/badge/React-0095D5?&style=for-the-badge&logo=react&logoColor=white) [![GitHub last commit](https://img.shields.io/github/last-commit/code-gambit/VT-WebClient?style=for-the-badge)](https://github.com/code-gambit/VT-WebClient/commits)
+
+## Project description
+This project is the web client for V Transfer, allowing people to share their file easily with minimum steps without compromising the security. At the backend we are using [IPFS](https://ipfs.io/) to upload a file which is a blockchain-based technology for file sharing making it secure and reliable, below are some of the key feature of this app.
```

Comment:
Add some pause in this line.
`At the backend we are using [IPFS](https://ipfs.io/), to upload a file, which is a blockchain-based technology for file sharing and thus making file upload more secure and reliable. Given below are some of the key feature of this app.
 `